src/api/che.ts

```diff
@@ -199,12 +199,15 @@ export class CheHelper {
     }
   }
 
-  async startWorkspace(cheNamespace: string, workspaceId: string, accessToken?: string): Promise<void> {
+  async startWorkspace(cheNamespace: string, workspaceId: string, debug: boolean, accessToken?: string): Promise<void> {
```

Comment:
Maybe we can make `debug` flag defaults to `false` ?

Reply:
Not much advantages